src/main/scala/lenthall/ErrorOr.scala

```diff
@@ -0,0 +1,22 @@
+package lenthall
+
+import cats.data.Validated.{Invalid, Valid}
+import cats.data.{NonEmptyList, Validated}
+
+object ErrorOr {
+  type ErrorOr[A] = Validated[NonEmptyList[String], A]
+
+  implicit class FlatMap[A](val fa: ErrorOr[A]) extends AnyVal {
+    /**
+      * Not consistent with `Applicative#ap` but useful in for comprehensions.
+      *
+      * @see http://typelevel.org/cats/tut/validated.html#of-flatmaps-and-xors
+      */
+    def flatMap[B](f: A => ErrorOr[B]): ErrorOr[B] = {
```

Comment:
I thought the idea behind `core` (or to put another way, what I intended when I initially split the subprojects and made one called `core`) was to be stuff that exactly fits this description. i.e. a catch-all for things which should be shared across subprojects.